src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processor/CourtCaseEventsProcessor.kt

```diff
@@ -48,18 +46,13 @@ class CourtCaseEventsProcessor(
 
       else -> {
         log.debug("Received case type ${UNKNOWN.name}")
-        telemetryService.trackEvent(UNKNOWN_CASE_RECEIVED, emptyMap())
       }
     }
   }
 
   fun processLibraHearingEvent(libraHearingEvent: LibraHearingEvent) {
     log.debug("Processing LIBRA event")
     val person = Person.from(libraHearingEvent)
-    telemetryService.trackEvent(
-      NEW_LIBRA_CASE_RECEIVED,
-      mapOf("PNC" to person.otherIdentifiers?.pncIdentifier.toString(), "CRO" to person.otherIdentifiers?.cro),
-    )
```

Comment:
We don't see a reason to differentiate between Libra and HMCTS message sources for AppInsights. We have log statements if we want to draw any conclusions from the message source.